config/locales/simple_form.en.yml

```diff
@@ -4,7 +4,7 @@ en:
     "no": "No"
     required:
       text: "required"
-      mark: '*'
+      mark: "#"
```

Comment:
A question: why "#" instead of a "*"?

Reply:
Just trying out different looks, none of them get rid of the ".." underneath the symbol. Changed back to "*" in sha: 439f3ff

Reply:
To get rid of the line underneath, in the stylesheets, define:
```
abbr[title] {
  text-decoration: none;
}
```